README.md

```diff
@@ -1,5 +1,167 @@
-# Reconstruction tools
+# Reconstruction pipeline
 
-Tools for building full brain network models from standard structural MR scans.
+This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.
+It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB. 
+
+The mandatory inputs are DWI and T1 scans. Optionally, CT scans can be given as input, if sensors preprocessing is needed.
+
+We are using the Pegasus WMS in order to connect and automatize the pipeline steps.
```

Comment:
Maybe add a link towards Pegasus official page?
And write in parentheses what license type Pegasus has.